pandas/tests/plotting/test_frame.py

```diff
@@ -2921,6 +2972,14 @@ def test_secondary_axis_font_size(self, method):
         self._check_ticks_props(axes=ax.right_ax,
                                 ylabelsize=fontsize)
 
+    def _assert_ytickslabels_visibility(self, axes, expected):
```

Comment:
you you move these to the very top of the class (I think we hvae other checker functions there)